### ISSUE TITLE
CoreCLR test tree build is very slow

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/coreclr#7916]
Building tests in CoreCLR now takes me around 1 hour. This seems excessive.
```
Build started 10/31/2016 2:14:57 PM.

    3302 Warning(s)
    0 Error(s)

Time Elapsed 01:00:05.25
```
Not sure of all the reasons, but among other things the test build creates a 4GB log file...
```
10/31/2016  03:15 PM     3,827,466,190 Tests_Managed_Windows_NT__x64__Checked.log
```

**Comment by AndyAyersMS** [comment]
Log verbosity is set to `diag` when building tests. Dropping this to `normal` cuts the log down to 20MB and dropts the test rebuild time to 8 minutes on my machine. This may not be an entirely fair comparison to the 1hr above; I'll do a clean test build at some point to double-check.

**Comment by danmosemsft** [comment]
This was introduced by @wtgodbe in 69ab16ac55d423935dd40c12237903b0c3a0c170.  Guess: it was accidentally left over from development.

Incidentally I think the best console verbosity is minimal. That should have been the default originally.

**Comment by danmosemsft** [comment]
fixed by  d3e9e00
